src/main/ipv-stub/ipv-token.ts

```diff
@@ -20,7 +25,13 @@ export const handler: Handler = async (
   }
 };
 
-function get(event: APIGatewayProxyEvent): APIGatewayProxyResult {
+async function get(
+  event: APIGatewayProxyEvent
+): Promise<APIGatewayProxyResult> {
+  const authCode = JSON.parse(event.body as string)["authCode"];
+  const userIdentity = await getUserIdentityWithAuthCode(authCode);
+  putUserIdentityWithToken("test-token", userIdentity);
+  deleteUserIdentityWithAuthCode(authCode);
```

Comment:
I'm not sure if we want to delete this right away. It should be stored with a 1 day TTL so deletion handled by AWS

Reply:
It was part of the ticket to delete straight after ? Unless it was referencing that it just should be deleted after 1 day.

Reply:
For now, I think it might be useful for debugging to not delete right away. Can review that once we've got a full journey working

Reply:
Added it to the list of things to discuss: https://govukverify.atlassian.net/browse/ATO-869

Reply:
Cool - happy then to just remove all these methods definitions and calls for now then.